compiler/tflchef/core/src/ModelChef.cpp

```diff
@@ -219,6 +189,20 @@ struct CookParams
   std::vector<std::string> &custom_code_vec;
   std::string noname;
 };
+#endif
+
+struct ModelChef
+{
+  std::unique_ptr<flatbuffers::FlatBufferBuilder> flatbuffer_builder;
+
+  std::vector<flatbuffers::Offset<::tflite::SignatureDef>> signdef_vec;
+  std::vector<flatbuffers::Offset<::tflite::Buffer>> buffer_vec;
+  std::vector<flatbuffers::Offset<::tflite::OperatorCode>> code_vec;
+  std::vector<flatbuffers::Offset<::tflite::SubGraph>> subgraph_vec;
+  std::map<tflite::BuiltinOperator, int32_t> builtin_code_map;
+  std::vector<std::string> custom_code_vec;
```

Comment:
Is there any specific reason to remove the reference indicator(`&`)?

```suggestion
  std::unique_ptr<flatbuffers::FlatBufferBuilder> &flatbuffer_builder;

  std::vector<flatbuffers::Offset<::tflite::SignatureDef>> &signdef_vec;
  std::vector<flatbuffers::Offset<::tflite::Buffer>> &buffer_vec;
  std::vector<flatbuffers::Offset<::tflite::OperatorCode>> &code_vec;
  std::vector<flatbuffers::Offset<::tflite::SubGraph>> &subgraph_vec;
  std::map<tflite::BuiltinOperator, int32_t> &builtin_code_map;
  std::vector<std::string> &custom_code_vec;
```

Reply:
> Is there any specific reason to remove the reference indicator(&)?

`CookParams` was to pass multiple parameters but
`ModelChef` will be used to hold these parameters as members.